Code/Framework/AzFramework/Platform/Common/Default/AzFramework/Process/ProcessWatcher_Default.cpp

```diff
@@ -83,4 +83,9 @@ namespace AzFramework
     {
 
     }
+
+    AZStd::string ProcessLauncher::ProcessLaunchInfo::GetCommandLineParametersAsString() const
+    {
+        return "";
```

Comment:
Actually it is slightly more efficient to return `AZStd;::string{}` than `""` to avoid the `strlen` call

Reply:
Updated